include/openenclave/attestation/verifier.h

```diff
@@ -154,7 +154,11 @@ oe_result_t oe_verifier_free_format_settings(uint8_t* settings);
  * address of the dynamically allocated list of claims (including base and
  * custom).
  * @param[out] claims_length If not NULL, the length of the claims list.
- * @retval OE_OK The function succeeded.
+ * @retval OE_OK The function succeeded, the claim OE_CLAIM_TCB_STATUS should be
+ * OE_TCB_STATUS_SGX_UP_TO_DATE or OE_TCB_STATUS_SGX_SW_HARDENING_NEEDED.
+ * @retval OE_TCB_LEVEL_INVALID The function succeeded, the claim
```

Comment:
"The TCB is not considered up-to-date, the claim OE_CLAIM_TCB_STATUS and OE_TCB_STATUS_SGX_UP_TO_DATE contain more information"